main/head.jsx

```diff
@@ -3,7 +3,7 @@ import NextHead from 'next/head'
 
 const Head = () => (
   <NextHead>
-    <title>CORE Search</title>
+    <title>CORE – Aggregating the world’s open access research papers</title>
```

Comment:
```suggestion
    <title>CORE – aggregating the world’s open access research papers</title>
```

Reply:
I used the same title as core.ac.uk hence capital A